src/main/java/com/conveyal/gtfs/error/NewGTFSErrorType.java

```diff
@@ -11,6 +11,7 @@
     URL_FORMAT(Priority.MEDIUM, "URL format should be <scheme>://<authority><path>?<query>#<fragment>"),
     LANGUAGE_FORMAT(Priority.LOW, "Language should be specified with a valid BCP47 tag."),
     ILLEGAL_FIELD_VALUE(Priority.MEDIUM, "Fields may not contain tabs, carriage returns or new lines."),
+    FIELD_VALUE_TOO_LONG(Priority.MEDIUM, "Field value has too many characters."),
```

Comment:
@binh-dam-ibigroup, can you add a comment above this new type describing that this is an MTC-specific issue type (for now). Another approach would be to add another Priority enum (e.g. `Priority.MTC`), but I'm not 100% on that approach.

cc @evansiroky

Reply:
I'll add the comment for MTC. I also think error types should remain generic if another agency wants to use them.